lib/activerecord-clean-db-structure/clean_dump.rb

```diff
@@ -66,8 +68,12 @@ def run
       partitioned_tables += dump.scan(partitioned_tables_regexp2).map(&:first)
 
       partitioned_tables.each do |partitioned_table|
-        partitioned_schema_name, partitioned_table_name_only = partitioned_table.split('.', 2)
+        _partitioned_schema_name, partitioned_table_name_only = partitioned_table.split('.', 2)
+
+        next if partitioned_table_name_only.starts_with?('storage_tables_blobs_partition_')
```

Comment:
Het lijkt me niet handig om namen van specifieke partitions hier op te moeten geven, anders moeten we voor iedere partition die we introduceren de gem updaten. Zou mooi zijn als dit automatisch goed gaat. En als het niet anders kan dat we een lijst kunnen opgeven van partitions die behouden moeten blijven.

Reply:
Klopt was inderdaad niet heel erg logisch, had ik me ook later bedacht. Het is nu configurable via de options.